client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -76,11 +76,11 @@ class Widget {
     tempContainer.innerHTML = html.trim();
     const dom = tempContainer.firstChild;
 
-    /* replace the placeholder with the new element */
-    placeholder.replaceWith(dom);
-
     /* execute any scripts in the new element */
-    runInlineScripts(dom);
+    runInlineScripts(tempContainer);
+
+    /* replace the placeholder with the new element */
+    placeholder.replaceWith(...tempContainer.childNodes);
```

Comment:
With this, we replace the placeholder with all the nodes in the HTML (that was put in the `div` `tempContainer`). However, we keep only passing the `tempContainer.firstChild` as the element to the bound widget class (see that the above `const dom = tempContainer.firstChild;` do not change).

We change the `runInlineScript` call from the `dom` (first child) to the `tempContainer` in case there are scripts at the top-level (as the case with #11935).

Then, we need to move the `runInlineScripts` call to _before_ the placeholder is swapped, because once it is, `tempContainer` will be empty. We can't do `runInlineScripts(dom.parentElement)` since we can't be sure that the parent does not have other scripts already in place.